drag-n-drop.js

```diff
@@ -96,6 +99,86 @@ H5P.DragNDrop.moveHandler = function (event) {
 H5P.DragNDrop.prototype.move = function (x, y) {
   var that = this;
```

Comment:
We should refrain from assigning variables to `this`. As far as I can see, there are no reason for using `that` instead of `this` anywhere in this method.

Reply:
Changed from 'that' to 'this' in move-function